flink/src/main/java/org/apache/iceberg/flink/sink/IcebergFilesCommitter.java

```diff
@@ -120,7 +123,10 @@ public void initializeState(StateInitializationContext context) throws Exception
     // Open the table loader and load the table.
     this.tableLoader.open();
     this.table = tableLoader.loadTable();
-
+    final Map<String, String> properties = table.properties();
+    maxContinuousEmptyCommits = PropertyUtil.propertyAsInt(properties, MAX_CONTINUOUS_EMPTY_COMMITS, 10);
+    Preconditions.checkArgument(maxContinuousEmptyCommits > 0, MAX_CONTINUOUS_EMPTY_COMMITS + " must not be " +
```

Comment:
As we will `MOD` the `maxContinuousEmptyCommits` in [here](https://github.com/apache/iceberg/pull/2042/files#diff-5320ef0a19dedc7a2693ccb49027baef1ad1a480df9d05688bbfe66410ce5bfcR214), so the `maxContinuousEmptyCommits` must be positive I think.  We may need to correct the error message in `Preconditions.checkArgument`

Reply:
ok,it's right